src/components/simple-billboard/index.js

```diff
@@ -83,30 +83,28 @@ const SimpleBillboard = ({ metric, statusTrend = {}, title }) => {
   }, [difference]);
 
   return (
-    <div>
```

Comment:
Not a biggie, but IMHO, it's best to always return a div for a component. At least for this component it doesn't make sense to return a fragment. Even if you have no styling on the div, I'd still do something like below, so when we inspect the code we know this was the simple billboard component.
```
<div className="simple-billboard">
```